saleor/graphql/product/types/products.py

```diff
@@ -1060,21 +1063,24 @@ def __resolve_reference(root, _info, **_kwargs):
 
 
 @key(fields="id")
-class ProductImage(CountableDjangoObjectType):
+class ProductMedia(CountableDjangoObjectType):
```

Comment:
We cannot remove `ProductImage` type yet. While it's fine to do this change for models, in API this is breaking change for all clients. We need to deprecate all queries returning `ProductImage` in Saleor 2.11. We will be able to remove this type and fields in Saleor 4.0.